package.json

```diff
@@ -13,5 +13,9 @@
   },
   "devDependencies": {
     "prettier": "^3.2.5"
+  },
+  "dependencies": {
+    "@vueuse/core": "^10.9.0",
+    "focus-trap": "^7"
```

Comment:
Wrong package

Reply:
@anbraten are you refering to the focus trap or the vueUse core? i had to add the focus trap package to be able to use it as a component

Reply:
@ShahwarZia The package.json file is wrong. It has to be added in the former package

Reply:
@anbraten done